---
title: Analyse génomique de la propagation du nCoV. Rapport de situation au 2020-01-25.
authors: "Trevor Bedford, Richard Neher, James Hadfield, Emma Hodcroft, Misja Ilcisin, Nicola Müller"
authorLinks: "https://nextstrain.org"
affiliations: "Fred Hutch, Seattle, États-Unis et Biozentrum, Bâle, Suisse"
date: "2020 Jan 30"
dataset: "https://nextstrain.org/ncov/2020-01-25?d=map"
abstract: "Ce rapport utilise des données génomiques du nouveau coronavirus (nCoV) rendues publiques sur GISAID et Genbank afin d'estimer la vitesse et les motifs de la propagation de l'épidémie virale. Nous prévoyons d'émettre des rapports de situation à mesure que de nouvelles données sont produites et partagées. Ce site web est optimisé pour être affiché sur un ordinateur de bureau."
---

# [Résumé exécutif](https://nextstrain.org/ncov/2020-01-25)

```auspiceMainDisplayMarkdown
## Résumé exécutif

Nous avons examiné la diversité génétique de 42</tag> génomes du nouveau coronavirus (nCoV) rendus publics pour inférer la date de l'ancêtre commun et le taux de propagation.
Principaux résultats :
* Les 42</tag> génomes sont très similaires, différant du consensus de 0 à 7 mutations.
* L'explication parcimonieuse de cette faible diversité est que l'épidémie est issue d'une unique introduction du virus dans la population humaine, ou d'un faible nombre de transmissions animal-homme de virus très similaires. 
* Cette transmission a probablement eu lieu en novembre ou début décembre 2019.  
* La transmission d'humain à humain a eu lieu en continu depuis cette date, aboutissant aux cas observés. 
* À l'aide des estimations du nombre total de cas de l'Imperial College de Londres, nous avons inféré un taux de reproduction entre 1,5 et 3,5, indiquant uns croissance rapide sur la période novembre-janvier. 
```

# [Coronavirus](https://nextstrain.org/ncov/2020-01-25)

### Lectures additionnelles:

* Informations générales sur le coronavirus sur [Wikipedia](https://fr.wikipedia.org/wiki/Coronavirus) _2020-01-30_
* Résumé de l'épidémie du nCoV sur [Wikipedia](https://fr.wikipedia.org/wiki/%C3%89pid%C3%A9mie_de_coronavirus_de_2019-2020) _2020-01-30_
* Documents fournis par l'[US CDC](https://www.cdc.gov/coronavirus/index.html) _2020-01-29_
* Caractéristiques du génome sur [ViralZone](https://viralzone.expasy.org/764?outline=all_by_species) _2020-01-23_
* Analyse interactive du risque par [MOBS-lab](https://datastudio.google.com/reporting/3ffd36c3-0272-4510-a140-39e288a9f15c/page/U5lCB) _2010-01-29_
* Analyse interactive du risque par [ROCS-lab](http://rocs.hu-berlin.de/corona/) _2010-01-29_

```auspiceMainDisplayMarkdown

## Différents coronavirus humains

Les coronavirus (CoV) regroupent diverses espèces de virus à ARN de sens positif ((+)ssRNA) causant des infections respiratoires chez l'Homme.
Certaines variantes de coronavirus sont associées à des épidémies là où d'autres sont en circulation continue et causent principalement des infections respiratoires légères (par exemple le rhume).

#### SARS-CoV et MERS-CoV
Le plus connu de ces coronavirus est le [SARS-CoV](https://fr.wikipedia.org/wiki/Syndrome_respiratoire_aigu_s%C3%A9v%C3%A8re_li%C3%A9_au_coronavirus) ("syndrome respiratoire aigu sévère"), responsable de l'épidémie mondiale entre novembre 2002 et juillet 2003 ayant causé [plus 8000 cas et 774 décès](https://www.theguardian.com/world/2017/dec/10/sars-virus-bats-china-severe-acute-respiratory-syndrome), soit un taux de mortalité de 9–11%.

En 2012, un nouveau coronavirus causant de graves symptômes respiratoires, [MERS-CoV](https://fr.wikipedia.org/wiki/Coronavirus_du_syndrome_respiratoire_du_Moyen-Orient) ("Middle East respiratory syndrome"), a été identifié. Le MERS a causé un nombre de décès comparable au SRAS, bien que son mode de transmission soit très différent. Alors que le SRAS se transmettait efficacement d'un humain à un autre, les infections par le MERS étaient généralement l'effet de zoonoses indépendantes (transmissions d'animal à l'Homme) depuis le chameau (voir [Dudas _et al._](https://elifesciences.org/articles/31257) pour plus d'informations). L'épidémie était donc intrinsèquement limitée et essentiellement restreinte à la péninsule arabique. 


#### Seasonal CoV

Cependant, tous les coronavirus ne sont pas aussi mortels que le SARS-CoV et le MERS-CoV.
Quatre coronavirus "saisonnaux" infectent habituellement l'humain chaque année.
Par rapport au SRAS, ces coronavirus saisonnaux sont ["bien plus prévalents, beaucoup moins graves, et sont une cause fréquente de syndromes grippaux"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5820427/).
En effet, [5](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2879166/)–[12](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5820427/)% de tous les syndromes grippaux sont testés positivement pour le coronavirus. Ils sont donc plutôt communs et causent des millions d'infections légères chaque année.
Les coronavirus saisonnaux sont le résultat de plusieurs zoonoses depuis le réservoir animal des chauves-souris vers les humains dans la dernière centaine d'années. Après ces contagions inter-espèces, chaque virus saisonnier s'est répandu et établi dans la population humaine.


#### Réservoirs animaux
Les coronavirus infectent a une grande diversité d'animaux, et les épidémies humaines décrites plus haut sont le résultat d'un ou plusieurs "sauts" de ces réservoirs animaux vers la population humaine.
On pense que le SRAS est arrivé dans la population humaine depuis [la chauves-souris rhinolophe par l'intermédiaire de la civette masquée](https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1006698).


#### Transmission de personne à personne
La capacité de différents lignages de se transmettre d'humain à humain est extrêmement importante pour comprendre le développement potentiel d'une épidémie. 
À cause de sa capacité à se propager chez l'humain et de son fort taux de mortalité, le SRAS (ou un virus semblable au SRAS) est considéré comme une [menace majeure à la santé publique](https://www.who.int/whr/2007/overview/fr/index1.html) par l'OMS.

```

# [Nouveau coronavirus (nCoV) 2019-2020](https://nextstrain.org/ncov/2020-01-25)

### Lectures additionnelles (en anglais):

* New China virus: Five questions scientists are asking  [Nature news](https://www.nature.com/articles/d41586-020-00166-6) _2020-01-22_
* China virus latest: first US case confirmed  [Nature news](https://www.nature.com/articles/d41586-020-00154-w) _2020-01-21_
* New virus surging in Asia rattles scientists  [Nature news](https://www.nature.com/articles/d41586-020-00129-x) _2020-01-20_
* New virus identified as likely cause of mystery illness in China [Nature news](https://www.nature.com/articles/d41586-020-00020-9) _2020-01-08_

```auspiceMainDisplayMarkdown

## Recent outbreak of a novel coronavirus
In December 2019, a new illness was first detected in Wuhan, China.
We now know this to be another outbreak of coronavirus in humans (the 7th), and it is provisionally being called nCoV (novel coronavirus).

As of January 30th over 7,914 cases and 170 deaths [have been reported](https://en.wikipedia.org/wiki/2019%E2%80%9320_outbreak_of_novel_coronavirus_(2019-nCoV)).
It's still too early to know the case fatality rate, but early indications are that it is significantly less than SARS-CoV.
The case counts are dramatically rising in part due to increased surveillance and testing.

While the outbreak seems to be centered in Wuhan, which is now [under quarantine](https://twitter.com/PDChina/status/1220060879112282117), the virus has spread throughout China and abroad, including Hong Kong, Singapore, Japan, and Thailand, as well as Europe, North America, South Asia, the Middle East, and Australia. Limited local transmission outside of China has been reported.

The origin of the virus is still unclear, however [genomic analyis](https://virological.org/t/ncovs-relationship-to-bat-coronaviruses-recombination-signals-no-snakes/331) suggests nCoV is most closely related to viruses previously identified in bats.
It is plausible that there were other intermediate animal transmissions before the introduction into humans.
There is no evidence of snakes as an intermediary.

#### Nextstrain narratives

The following pages contain analysis performed using [Nextstrain](https://nextstrain.org).
Scrolling through the left hand sidebar will reveal paragraphs of text with a corresponding visualization of the genomic data on the right hand side.

To have full genomes of a novel and large RNA virus this quickly is a remarkable achievement.
These analyses have been made possible by the rapid and open sharing of genomic data and interpretations by scientists all around the world (see the final slide for a visualization of sequencing authorship).
```

# [How to interpret the phylogenetic trees](https://nextstrain.org/ncov/2020-01-30)

### Further Reading:

* [Exploring interactive phylogenies with Auspice](https://neherlab.org/201901_krisp_auspice.html) _2019-01-24_

```auspiceMainDisplayMarkdown
## Transmission trees vs phylogenetic trees

Pathogens spread through rapid replication in one host followed by transmission to another host.
An epidemic can only take off when one infection results in more than one subsequent infections.

As the pathogen replicates and spreads, its genome needs to be replicated many times and random mutations (copying mistakes)  will accumulate in the genome.
Such random mutations can help to track the spread of the pathogen and learn about its transmission routes and dynamics.

<div>
  <img alt="cartoon showing how transmission tree and phylogenetic tree relate" width="500" src="https://neherlab.org/talk_images/infection_tree_combined.png"/>
</div>

The illustration above shows a sketch of a transmission tree with a subset of cases that were sampled (blue).
In practice, the transmission tree is unknown and typically only rough estimates of case counts are available.
Genome sequences allow us to infer parts of the transmission tree.
In this example, three mutations (little diamonds) are indicated on the tree.
Sequences that have the same mutations are more closely related, so these mutations allow us to group samples into clusters of closely related viruses that belong to the same transmission chains.

### Reading a Phylogenetic Tree

Below, we see an illustration with a phylogenetic tree on the left, where mutations are shown as colored circles. On the right are the corresponding sequences, also with mutations shown as colored circles.
We can see that sequences that share the same mutations group together.
When sequences appear linked by a flat vertical line, like A and B, this means there are no differences between them – their sequences are identical.

When a sequence sits on a long line on its own, like C or E, this means it has unique mutations not found in other sequences. The longer the line, the more mutations.
A and B also have unique mutations (the green circle) not shared by the other sequences, but they are identical to each other.

<div>
  <img alt="cartoon of phylogenetic tree and corresponding alignment, with samples labelled A-E" width="500" src="https://data.nextstrain.org/toy_alignment_tree.png"/>
</div>

At the moment, the novel coronavirus (nCoV) phylogeny may not look much like a 'tree'.
Many of the sequences are identical – they sit together on vertical lines like A and B (some are on the left-most part of the tree).
Others have unique or shared mutations and so sit on lines, or 'branches', going to the right.
You can see how many mutations a branch has by hovering your mouse over it.
```

# [Phylogenetic analysis](https://nextstrain.org/ncov/2020-01-30?m=div&d=tree)

Here we present a phylogeny of 42</tag> strains of nCoV that have been publicly shared.
Information on how the analysis was performed is available [in this GitHub repository](github.com/nextstrain/ncov).

<br>

The colours represent the within-country region or US-state of isolation, with the x-axis representing nucleotide divergence.

<br>

Divergence is measured as the number of changes (mutations) in the genome.
Several sequences have zero mutations -- meaning they are all identical to the root (center) of the tree.
Other viruses have between one and seven mutations.

<br>

Sequencing the genome of a large novel RNA virus in an evolving outbreak situation is challenging.
Some of the differences observed in these sequences may be sequencing errors rather than actual mutations.
Insertions, deletions, and differences at the ends of the genome are more likely to be errors and so we masked these for the purposes of this analysis.

# [Phylogenetic Interpretation](https://nextstrain.org/ncov/2020-01-30?m=div&d=tree)

We currently see little genetic diversity across the nCoV sequences, with 11</tag> out of 42</tag> sequences having no unique mutations.

<br>

Low genetic diversity across these sequences suggests that the most recent common ancestor of all nCoV sequences was fairly recent, since mutations accumulate slowly compared to other RNA viruses at a rate of around 1-2 mutations per month for coronaviruses.
Generally, repeated introductions from an animal reservoir will show significant diversity (this has been true for Lassa, Ebola, MERS-CoV and avian flu).
The observation of such strong clustering of human infections can be explained by an outbreak that descends from a single zoonotic introduction event into the human population followed by human-to-human epidemic spread.

<br>

We are starting to see groups of sequences that share mutations.
One cluster contains sequences from Guangdong and four isolates from the US.
Other clusters contain two to four isolates.
Sequences in these clusters tend to be from more recent samples, suggesting that the virus has started to accumulate mutations as it spread in Wuhan and subsequently to other cities.
There is currently no evidence that these mutations change how the virus behaves -- it is expected that RNA viruses mutate.

# [Within-family transmission 1](https://nextstrain.org/ncov/2020-01-30?m=div&d=tree&f_location=Zhuhai)

There are three genetically-identical isolates from Zhuhai (Southeastern China, Guangdong Province) which form a cluster, sharing one unique mutation seen in no other isolate (you can hover your mouse over the branches to see which mutations are present).

<br>

Two of these cases (ending 028 and 040) are [known to come from a single family](https://twitter.com/JingLu_LuJing/status/1220143773532880896), again indicating human-to-human transmission.
We don't have information about the third case.


# [Within-family transmission 2](https://nextstrain.org/ncov/2020-01-30?m=div&d=tree&f_location=Shenzhen)

Of the six isolates from Guangdong Province (which includes the city of Shenzhen) we see four isolates which are genetically identical.
These sequences differ by 3 mutations from the root of the tree.

<br>

Three of the sequences from Guangdong (ending F025, F013, and F012) are [known to come from a single family](https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(20)30154-9/fulltext), and almost certainly represent human-to-human transmission.


<br>

# [Within-family transmission 2 - shared mutations](https://nextstrain.org/ncov/2020-01-30?m=div&d=tree&f_location=Shenzhen,Los%20Angeles,Orange%20County,Seattle,Chicago,Phoenix)

The three mutations found in this cluster are also present in the Arizona, USA isolate, and two of the mutations are found in three other USA isolates.


<br>

# [Within-family transmission 3](https://nextstrain.org/ncov/2020-01-30?m=div&d=tree&f_location=Paris)

Finally, the two sequences from France are identical, sharing one unique mutation, and one mutation also found in one of the USA isoaltes and the Taiwanese isolate.

<br>

The two french sequences are [known to be from the same family](https://www.thelocal.fr/20200129/coronavirus-in-france-what-you-need-to-know) - a Chinese couple from Wuhan.


# [Cases outside China](https://nextstrain.org/ncov/2020-01-30?c=country&d=tree&m=div)

There are reported diagnostically confirmed nCoV cases in many East and South-East Asian countries, USA, Australia, the Middle East, and Europe.
Vietnam, Japan, and Germany have reported transmission within the country, albeit always with a known link to Wuhan, China.

<br>

The only currently available sequence data for cases outside of China are the two cases from Thailand, five from the USA, two from France, and one from Taiwan.
The Thai samples are genetically identical to nine Chinese sequences, including seven isolated in Wuhan.
Four sequences from the US share two mutations with the cluster of sequences from Shenzhen.
The remaining sequence from the US shares a mutation with the sequence from Taiwan and the two from France.

<br>

The most parsimonious explanation for the observed pattern of mutation sharing between the US and Shenzhen sequences is a virus variant with the two shared mutations was circulating in Wuhan and was independently exported to Shenzhen and multiple times to the USA.
There is no evidence for a link between US sequences other than a link to Wuhan.

# [Dating the time of the most recent common ancestor](https://nextstrain.org/ncov/2020-01-30?d=tree)
The high similarity of the genomes suggests they share a recent common ancestor (i.e. that they have descended from the same ancestral virus recently). Otherwise, we would expect a higher number of differences between the samples.

<br>

Previous research on related coronavirus suggests that these viruses accumulate between 1 and 3 changes in their genome per month (rates of 3 &times; 10<sup>-4</sup> to 1 &times; 10<sup>-3</sup> per site per year).

<br>

On the right, we explore how different assumptions about the rate of change, and the observed genetic diversity, give us estimates for the timing of the outbreak.

```auspiceMainDisplayMarkdown
## Date of the common ancestor of outbreak viruses
With the additional sequences shared during the past week, the tree now shows several distinct clusters such that our analysis from 2020-01-25 assuming a star-like topology is no longer appropriate.

We reproduce here our analysis based on data available up to 2020-01-25
assuming a star-like phylogeny structure along with a Poisson distribution of mutations through time to estimate the time of the most recent common ancestor ('TMRCA') of sequenced viruses.
**We found that the common ancestor most likely existed between mid-Nov and the beginning of Dec 2019. The biggest source of uncertainty is the substitution rate.**

<div>
  <img alt="graph of TMRCA estimates based on different mutation rates" width="500" src="https://data.nextstrain.org/ncov_poisson-tmrca.png"/>
</div>

Using the entire data set, the nextstrain analysis pipeline estimates that the common ancestor most likely existed between late-Nov and the beginning of Dec 2019.

There is a [confirmed case in Wuhan with onset date of Dec 1, 2019](https://twitter.com/trvrb/status/1220749265380593664), which would put an upper bound on the date of most recent common ancestor.
The common ancestor of viruses sequenced to date might be later than this date though.

More detailed modeling of the onset of the outbreak are onging.
Despite considerable uncertainty, our best guess is remains late November/early December.

```

# [Estimating the growth rate](https://nextstrain.org/ncov/2020-01-30?d=tree)

An important quantity in the spread of a pathogen is the average number of secondary cases each infection produces.

<br>

This number is known as R0 ("R-zero" or "R-nought").
One the right, we present simple estimates of R0.

```auspiceMainDisplayMarkdown
## Estimates of epidemic growth rate
Scientists at Imperial College London have used the number of cases observed outside of China to estimate the [total number of cases](https://www.imperial.ac.uk/mrc-global-infectious-disease-analysis/news--wuhan-coronavirus/) and suggested that there have been at least several thousand cases by 2020-01-22.
With the additional exported cases since and the continued growth of confirmed cases in China, we currently have to expect at least 50000 cases to date.
Together with our previous estimates of the age of the outbreak and information on the infectious period, we can estimate plausible ranges of R0 using a branching process model.

**We find plausible estimates of R0 between 1.8 and 3.5.**

If we assume the outbreak started at the beginning of November 2019 (12 weeks ago), we find that R0 should range between 1.8 and 2.5, depending on how large ('n') the outbreak is now.
<div>
  <img alt="graph of R0 estimates with epidemic start 12 weeks ago" width="500" src="https://data.nextstrain.org/ncov_branching-R0-early_2020-01-29.png"/>
</div>

If we assume a more recent start, at the beginning of December 2019 (8 weeks ago), the estimates for R0 range between 2.2 and 3.5:
<div>
  <img alt="graph of R0 estimates with epidemic start 8 weeks ago" width="500" src="https://data.nextstrain.org/ncov_branching-R0-recent_2020-01-29.png"/>
</div>
This estimates are broadly consistent with those by other scientists which mostly fall between R0=2-3, see for example <a href="https://www.biorxiv.org/content/10.1101/2020.01.25.919787v1">this preprint</a>.
Importantly, R0 is a quantitity that depends strongly on the socio-economic context and infection control measures.
```

# [Scientific credit](https://nextstrain.org/ncov/2020-01-30?d=map&c=author)

We would like to acknowledge the amazing and timely work done by all scientists involved in this outbreak, but particularly those working in China.
Only through the rapid sharing of genomic data and metadata are analyses such as these possible.

<br>

The nCoV genomes were generously shared by scientists at the:

 * Shanghai Public Health Clinical Center & School of Public Health, Fudan University, Shanghai, China
 * National Institute for Viral Disease Control and Prevention, China CDC, Beijing, China
 * Institute of Pathogen Biology, Chinese Academy of Medical Sciences & Peking Union Medical College, Beijing, China
 * Wuhan Institute of Virology, Chinese Academy of Sciences, Wuhan, China
 * Department of Microbiology, Zhejiang Provincial Center for Disease Control and Prevention, Hangzhou, China
 * Guangdong Provincial Center for Diseases Control and Prevention
 * Department of Medical Sciences, National Institute of Health, Nonthaburi, Thailand
 * Division of Viral Diseases, Centers for Disease Control and Prevention, USA
 * Centers for Disease Control, R.O.C., Taipei, Taiwan
 * Institut Pasteur, Paris, France

# [Detailed scientific credit](https://nextstrain.org/ncov/2020-01-30?d=map&c=author)

These data were shared via [GISAID](https://gisaid.org).
We gratefully acknowledge their contributions.

<br>

To the right we give specific sequences shared by each lab.

```auspiceMainDisplayMarkdown

The nCoV genomes were generously shared by scientists at the

 * Shanghai Public Health Clinical Center & School of Public Health, Fudan University, Shanghai, China
   - Wuhan-Hu-1/2019
 * National Institute for Viral Disease Control and Prevention, China CDC, Beijing, China
   - Wuhan/IVDC-HB-01/2019
   - Wuhan/IVDC-HB-04/2020
   - Wuhan/IVDC-HB-05/2019)
 * Institute of Pathogen Biology, Chinese Academy of Medical Sciences & Peking Union Medical College, Beijing, China
   - Wuhan/IPBCAMS-WH-01/2019
   - Wuhan/IPBCAMS-WH-02/2019
   - Wuhan/IPBCAMS-WH-03/2019
   - Wuhan/IPBCAMS-WH-04/2019
 * Wuhan Institute of Virology, Chinese Academy of Sciences, Wuhan, China
   - Wuhan/WIV02/2019
   - Wuhan/WIV04/2019
   - Wuhan/WIV05/2019
   - Wuhan/WIV06/2019
   - Wuhan/WIV07/2019
 * Department of Microbiology, Zhejiang Provincial Center for Disease Control and Prevention, Hangzhou, China
   - Zhejiang/WZ-01/2020
   - Zhejiang/WZ-02/2020
 * Guangdong Provincial Center for Diseases Control and Prevention
   - Guangdong/20SF001/2020
   - Guangdong/20SF012/2020
   - Guangdong/20SF013/2020
   - Guangdong/20SF014/2020
   - Guangdong/20SF025/2020
   - Guangdong/20SF028/2020
   - Guangdong/20SF040/2020
   - Guangdong/20SF174/2020
   - Guangdong/20SF206/2020
   - Guangdong/20SF207/2020
   - Foshan/20SF207/2020
   - Foshan/20SF210/2020
   - Foshan/20SF211/2020

 * Department of Medical Sciences, National Institute of Health, Nonthaburi, Thailand
   - Nonthaburi/61/2020
   - Nonthaburi/74/2020
 * Division of Viral Diseases, Centers for Disease Control and Prevention, USA
   - USA-WA1/2020
   - USA/AZ1/2020
   - USA/IL1/2020
   - USA/CA1/2020
   - USA/CA2/2020
 * Centers for Disease Control, R.O.C., Taipei, Taiwan
   - Taiwan/2/2020
 * Institut Pasteur, Paris, France
   - France/IDF0372/2020
   - France/IDF0373/2020

```
