---
title: Genomic analysis of nCoV spread. Situation report 2020-01-25.
authors: "Trevor Bedford, Richard Neher, James Hadfield, Emma Hodcroft, Misja Ilcisin, Nicola Müller"
authorLinks: "https://nextstrain.org"
affiliations: "Fred Hutch, Seattle, USA and Biozentrum, Basel, Switzerland"
date: "2020 Jan 25"
dataset: "https://nextstrain.org/ncov/2020-01-25?d=map"
abstract: "这份报告使用了来自GISAID和Genbank公开共享的2019新型冠状病毒（nCoV）基因组数据来估计疫情的流传速度和方式。 一旦收集到新的病毒数据，我们将陆续更新该状况报告。请用桌面浏览器游览该网站"
---

# [Executive summary](https://nextstrain.org/ncov/2020-01-25)

```auspiceMainDisplayMarkdown
## Executive summary

至今我们使用27个公开共享的新型冠状病毒基因组，检查了遗传多样性以推断共同祖先的日期和传播速度。
我们发现:
* 27个采样的基因组非常相似，相差0-5个突变。 
* 缺乏遗传多样性表明了病毒可能来自于一次动物传人的病例或少数但相同的动物传人的病毒而引起的.  
* 此传播事件最有可能发生在2019年11月或12月初。 
* 传播事件之后，疫情即便开始人传人，导致观察到病例。 
* 根据伦敦帝国理工学院（Imperial London London）的总病例数估算值，我们得出的生殖数在1.5到3.5之间，表明在11月至1月期间迅速增长。 

```

# [冠状病毒](https://nextstrain.org/ncov/2020-01-25)

### 更多资料:

* 冠状病毒摘要[维基百科](https://en.wikipedia.org/wiki/Coronavirus) _2020-01-25_
* 新型冠状病毒爆发摘要 [维基百科](https://en.wikipedia.org/wiki/2019%E2%80%9320_Wuhan_coronavirus_outbreak) _2020-01-25_
* [美国CDC提供的材料](https://www.cdc.gov/coronavirus/index.html) _2020-01-23_
* 冠状病毒组织和基因组 [ViralZone](https://viralzone.expasy.org/764?outline=all_by_species) _2020-01-23_

```auspiceMainDisplayMarkdown

## 不同的人类冠状病毒

冠状病毒（CoV）是一种能引起呼吸道感染的正链单股RNA((+)ssRNA)病毒 。
某些冠状病毒变体能引发致命的呼吸系统疾病暴发，另一些则能持续传播并引起一般轻微感染症状（例如普通感冒）。

#### SARS-CoV 和 MERS-CoV
冠状病毒中最著名的是[严重急性呼吸道综合征冠状病毒(SARS-CoV)](https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome)。该病毒在2002年11月至2003年7月爆发，传播扩散于世界各地，导致[超过8000个病例和774例死亡](https://www.theguardian.com/world/2017/dec/10/sars-virus-bats-china-severe-acute-respiratory-syndrome)，病死率约为9-11％。

[中东呼吸综合征冠状病毒(MERS-CoV)](https://en.wikipedia.org/wiki/Middle_East_respiratory_syndrome)则是在2012年首次被确认。 虽然MERS-CoV和SARS-CoV导致的死亡率相当，它们的传播途径却大有差异。 SARS-CoV能有效地人传人，但MERS-CoV病例大多源自于人与动物（骆驼）接触的独立案件（人畜共患病）（请参阅[Dudas _et al._](https://elifesciences.org/articles/31257)了解详情）。这导致了一次局限于阿拉伯半岛的限制性爆发。 

#### 季节性冠状病毒
并非所有冠状病毒都像SARS-CoV和MERS-CoV一样致命。 
每年普遍感染人类的 “季节性”冠状病毒有四种。与SARS相比，这些季节性冠状病毒[“更为普遍，轻微，并且经常导致流感样疾病”](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5820427/)。
实际上，[5](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2879166/)–[12](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5820427/)%所呈现的流感样疾病都属阳性冠状病毒。因此冠状病毒是相当普遍的 - 每年导致数百万例轻微病例。这些季节性冠状病毒是在过去约100年前个别从蝙蝠传染给人类的。之后，该季节性病毒便在人群中广泛传播，独立演化。

#### 天然动物宿主  
冠状病毒可感染多种动物，而上述的人类冠状病毒可来至于单方或多方的天然动物宿主 – SARS病毒的[天然宿主为中华菊头蝠，并而透过果子狸为中间宿主](https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1006698)传染给人类。 

#### 人与人之间的传播
分析不同冠状病毒传染人类的能力对于了解疫情的爆发潜能和发展极为重要。由于SARS具有人与人之间传播的能力以及较高的病死率，SARS（或类似SARS的病毒）因此被视为[全球公共卫生威胁](https://www.who.int/whr/2007/overview/en/index1.html)。

```

# [2019-2020 新型冠状病毒](https://nextstrain.org/ncov/2020-01-25)

### 更多资料:

* New China virus: Five questions scientists are asking  [Nature news](https://www.nature.com/articles/d41586-020-00166-6) _2020-01-22_
* China virus latest: first US case confirmed  [Nature news](https://www.nature.com/articles/d41586-020-00154-w) _2020-01-21_
* New virus surging in Asia rattles scientists  [Nature news](https://www.nature.com/articles/d41586-020-00129-x) _2020-01-20_
* New virus identified as likely cause of mystery illness in China [Nature news](https://www.nature.com/articles/d41586-020-00020-9) _2020-01-08_

```auspiceMainDisplayMarkdown

## 最新爆发的新型冠状病毒
2019年12月，中国武汉首次发现新疾病。我们现以知道这是新一次的人类冠状病毒爆发（第7次），暂且称为新型冠状病毒。

截至1月25日，[已有1400多起案例和42例死亡](https://en.wikipedia.org/wiki/2019%E2%80%9320_outbreak_of_novel_coronavirus_(2019-nCoV)). 现在断定病死率还为时过早，但早期迹象表明它大大低于SARS-CoV。 
案件数量还在急剧增加的部分原因在于更广泛的侦察与测试。 

虽然疫情集中在武汉（[现已被隔离](https://twitter.com/PDChina/status/1220060879112282117)），该病毒已在中国和海外蔓延，包括香港，澳门，泰国， 日本，韩国，美国和法国。中国境外的本地传播暂时有限。

病毒的来源尚未清楚，但是[基因组分析](https://virological.org/t/ncovs-relationship-to-bat-coronaviruses-recombination-signals-no-snakes/331)表明新型冠状病毒与先前在蝙蝠中发现的冠状病毒基因最接近。 
暂时还不能排除有中间宿主的可能性，但没有证据表明蛇是中间宿主。 

#### Nextstrain 叙述

以下页面包含使用[Nextstrain](https://nextstrain.org)执行的分析。
滚动左侧边栏将显示文本段落，并在右侧显示相应的基因组数据视化。

快速破译新的大型RNA病毒完整基因组是一项了不起的成就。
这些分析能实现是因为世界各地的科学家快速开放地共享基因组数据和分解（有关基因组数据作者身份，请参见最后一张幻灯片）。

```

# [如何解读系统树](https://nextstrain.org/ncov/2020-01-25)

### 更多资料:

* [Exploring interactive phylogenies with Auspice](https://neherlab.org/201901_krisp_auspice.html) _2019-01-24_

```auspiceMainDisplayMarkdown
## 传播树与系统发生树

病原体能通过在一个宿主中快速复制然后传播到另一宿主。一宗病例唯有在逐渐传播后才能演变成传染行疾病。 

随着病原体的复制和传播，随机突变（复制错误）会在基因组中累积。 
这种随机突变可以帮助追踪病原体的传播并了解其传播途径和动态。 

<div>
  <img alt="cartoon showing how transmission tree and phylogenetic tree relate" width="500" src="https://neherlab.org/talk_images/infection_tree_combined.png"/>
</div>

上方的插图显示了传播树的示意图，其中包含抽样的案例（蓝色）。 
在实践中，传播树是未知的，只能粗略估计案例计数。 
基因组序列能让我们推断出传播树的一部分。 
在此示例中，树上指示了三个突变（小菱形）。 
具有相同突变的序列更紧密相关。因此这些突变能让我们揣测属于相同传播链的紧密相关病毒簇。 

### 解读系统树

以下左侧显示系统进化树的插图，其中的突变陈列为彩色圆圈。 右边是相应的序列，突变也陈列为彩色圆圈。 
我们可以看到，共享相同突变的序列会组合在一起。 
当序列显示为一条垂直的平线时（例如A和B），这意味着它们之间没有区别 - 它们的序列是相同的。

当一个序列像C或E那样排成一行时，这意味着它具有在其他序列中找不到的独特突变。 行越长，突变越多。
A和B也具有其他序列不共有的独特突变（绿色圆圈），但彼此相同。

<div>
  <img alt="cartoon of phylogenetic tree and corresponding alignment, with samples labelled A-E" width="500" src="https://nextstrain-data.s3.amazonaws.com/toy_alignment_tree.png"/>
</div>

目前，新型冠状病毒的系统树可能看起来并不像“树”。 
许多序列是相同的 – 它们像A和B一样排列在垂直线上（有些在树的最左侧）。 
其它则具有独特或共享的变异，因此排列于一行往右边 “分支”。 
您可以通过鼠标悬停在某个分支上来查看该分支有多少个突变。 

```

# [系统树分析](http://localhost:4000/ncov/2020-01-25?m=div&d=tree)

这里我们呈现公开共享的27种新型冠状病毒的系统树。 [GitHub存储库](github.com/nextstrain/ncov)中提供了有关如何执行分析的资料。

<br>

颜色表示病毒的起源地（隔别的城市/州），x轴表示核苷酸差异。 

<br>

散度测量为基因组中变化（突变）的数量。 
几个序列的突变为零，这意味着它们都与树根（中心）相同。 其他病毒具有一到五个突变。 

<br>

随着疫情不断地突变发展，破译新的大型RNA病毒基因组进行测序是具有大大挑战性的。这些序列中观察到的某些差异可能是测序错误，而不是实际的突变。 基因组末端的插入，缺失和差异更有可能是错误。出于分析目的，我们将其屏蔽。 

# [Phylogenetic Interpretation](https://nextstrain.org/ncov/2020-01-25?m=div&d=tree)

We currently see little genetic diversity across the nCoV sequences, with 8 out of 27 sequences having no unique mutations.

<br>

Low genetic diversity across these sequences suggests that the most recent common ancestor of all nCoV sequences was fairly recent, since mutations accumulate slowly compared to other RNA viruses at a rate of around 1-2 mutations per month for coronaviruses.
Generally, repeated introductions from an animal reservoir will show significant diversity (this has been true for Lassa, Ebola, MERS-CoV and avian flu).
The observation of such strong clustering of human infections can be explained by an outbreak that descends from a single zoonotic introduction event into the human population followed by human-to-human epidemic spread.

<br>

At the moment, most mutations that can be observed are singletons – they are unique to individual genomes. Only the sequences that form the two clusters from Guangdong and the sequence form the USA share mutations – we will explore these in later slides.

# [Within-family transmission 1](https://nextstrain.org/2020-01-25/ncov?m=div&d=tree&f_location=Zhuhai)

There are two genetically-identical isolates from Zhuhai (Southeastern China, Guangdong Province) which form a cluster, sharing one unique mutation seen in no other isolate (you can hover your mouse over the branches to see which mutations are present).

<br>

These two cases are [known to come from a single family](https://twitter.com/JingLu_LuJing/status/1220143773532880896), again indicating human-to-human transmission.


# [Within-family transmission 2](https://nextstrain.org/ncov/2020-01-25?m=div&d=tree&f_location=Shenzhen)

Of the six isolates from Shenzhen (Southeastern China, Guangdong Province) we see four isolates which are genetically identical.
These sequences differ by 3 mutations from the root of the tree.
Two of these mutations are also found in the sequence of the case in Washington, USA (more below).
One sequence from Shenzhen has two additional mutations.

<br>

The sequences form Shenzhen are [known to come from a single family](https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(20)30154-9/fulltext), and almost certainly represent human-to-human transmission.

<br>


# [Cases outside China](https://nextstrain.org/ncov/2020-01-25?c=country&d=tree&m=div)

There are reported diagnostically confirmed nCoV cases in many East and South-East Asian countries, USA, Australia, and Europe.
Vietnam has reported a case without travel history to Wuhan who possibly got infected from a [relative visiting from Wuhan](https://www.aljazeera.com/news/2020/01/china-coronavirus-outbreak-latest-updates-200123133559818.html).

<br>

The only currently available sequence data for cases outside of China are the two cases from Thailand and one case from the USA, which are coloured here in yellow and red, respectively.
The Thai samples are genetically identical to six Chinese sequences, including five isolated in Wuhan.
The sequence from Washington, US, shares two mutations with the cluster of sequences from Shenzhen.

<br>

The case from Washington, USA, had travelled to Wuhan.
The most parsimonious explanation for the observed pattern of mutation sharing is the virus variant with the two shared mutations was circulating in Wuhan and was independently exported to Shenzhen and to the USA.


# [Dating the time of the most recent common ancestor](https://nextstrain.org/ncov/2020-01-25?d=tree)
The high similarity of the genomes suggests they share a recent common ancestor (i.e. that they have descended from the same ancestral virus recently). Otherwise, we would expect a higher number of differences between the samples.

<br>

Previous research on related coronavirus suggests that these viruses accumulate between 1 and 3 changes in their genome per month (rates of 3 &times; 10<sup>-4</sup> to 1 &times; 10<sup>-3</sup> per site per year).

<br>

On the right, we explore how different assumptions about the rate of change, and the observed genetic diversity, give us estimates for the timing of the outbreak.

```auspiceMainDisplayMarkdown
## Date of the common ancestor of outbreak viruses
Here, we assume a star-like phylogeny structure along with a Poisson distribution of mutations through time to estimate the time of the most recent common ancestor ('TMRCA') of sequenced viruses.
**We find that the common ancestor most likely existed between mid-Nov and the beginning of Dec 2019.**

<div>
  <img alt="graph of TMRCA estimates based on different mutation rates" width="500" src="https://nextstrain-data.s3.amazonaws.com/ncov_poisson-tmrca.png"/>
</div>

As the more samples are sequenced, we expect the tree to show more structure, such that the star-like phylogeny topology is no longer a good assumption.
At this point, phylodynamic estimates of the age of the epidemic will become feasible.

There is a [confirmed case in Wuhan with onset date of Dec 1, 2019](https://twitter.com/trvrb/status/1220749265380593664), which would put an upper bound on the date of most recent common ancestor. And strongly suggests evolutionary rates of ~3.0 x 10^-4 are more plausible.

```

# [Estimating the growth rate](https://nextstrain.org/ncov/2020-01-25?d=tree)

An important quantity in the spread of a pathogen is the average number of secondary cases each infection produces.

<br>

This number is known as R0 ("R-zero" or "R-nought").
One the right, we present simple estimates of R0.

```auspiceMainDisplayMarkdown
## Estimates of epidemic growth rate
Scientists at Imperial College London have used the number of cases observed outside of China to estimate the [total number of cases](https://www.imperial.ac.uk/mrc-global-infectious-disease-analysis/news--wuhan-coronavirus/) and suggested that there have been at least several thousand cases.
Together with our previous estimates of the age of the outbreak and information on the infectious period, we can estimate plausible ranges of R0 using a branching process model.

**We find plausible estimates of R0 between 1.5 and 3.5.**

If we assume the outbreak started at the beginning of November 2019 (12 weeks ago), we find that R0 should range between 1.5 and 2.5, depending on how large ('n') the outbreak is now.
<div>
  <img alt="graph of R0 estimates with epidemic start 12 weeks ago" width="500" src="https://nextstrain-data.s3.amazonaws.com/ncov_branching-R0-early.png"/>
</div>

If we assume a more recent start, at the beginning of December 2019 (8 weeks ago), the estimates for R0 range between 1.8 and 3.5:
<div>
  <img alt="graph of R0 estimates with epidemic start 8 weeks ago" width="500" src="https://nextstrain-data.s3.amazonaws.com/ncov_branching-R0-recent.png"/>
</div>
```

# [Scientific credit](https://nextstrain.org/ncov/2020-01-25?d=map&c=author)

We would like to acknowledge the amazing and timely work done by all scientists involved in this outbreak, but particularly those working in China.
Only through the rapid sharing of genomic data and metadata are analyses such as these possible.

<br>

The nCoV genomes were generously shared by scientists at the:

 * Shanghai Public Health Clinical Center & School of Public Health, Fudan University, Shanghai, China
 * National Institute for Viral Disease Control and Prevention, China CDC, Beijing, China
 * Institute of Pathogen Biology, Chinese Academy of Medical Sciences & Peking Union Medical College, Beijing, China
 * Wuhan Institute of Virology, Chinese Academy of Sciences, Wuhan, China
 * Department of Microbiology, Zhejiang Provincial Center for Disease Control and Prevention, Hangzhou, China
 * Guangdong Provincial Center for Diseases Control and Prevention
 * Department of Medical Sciences, National Institute of Health, Nonthaburi, Thailand
 * Division of Viral Diseases, Centers for Disease Control and Prevention, USA

# [Detailed scientific credit](https://nextstrain.org/ncov/2020-01-25?d=map&c=author)

These data were shared via [GISAID](https://gisaid.org).
We gratefully acknowledge their contributions.

<br>

To the right we give specific sequences shared by each lab.

```auspiceMainDisplayMarkdown

The nCoV genomes were generously shared by scientists at the

 * Shanghai Public Health Clinical Center & School of Public Health, Fudan University, Shanghai, China
   - Wuhan-Hu-1/2019
 * National Institute for Viral Disease Control and Prevention, China CDC, Beijing, China
   - Wuhan/IVDC-HB-01/2019
   - Wuhan/IVDC-HB-04/2020
   - Wuhan/IVDC-HB-05/2019)
 * Institute of Pathogen Biology, Chinese Academy of Medical Sciences & Peking Union Medical College, Beijing, China
   - Wuhan/IPBCAMS-WH-01/2019
   - Wuhan/IPBCAMS-WH-02/2019
   - Wuhan/IPBCAMS-WH-03/2019
   - Wuhan/IPBCAMS-WH-04/2019
 * Wuhan Institute of Virology, Chinese Academy of Sciences, Wuhan, China
   - Wuhan/WIV02/2019
   - Wuhan/WIV04/2019
   - Wuhan/WIV05/2019
   - Wuhan/WIV06/2019
   - Wuhan/WIV07/2019
 * Department of Microbiology, Zhejiang Provincial Center for Disease Control and Prevention, Hangzhou, China
   - Zhejiang/WZ-01/2020
   - Zhejiang/WZ-02/2020
 * Guangdong Provincial Center for Diseases Control and Prevention
   - Guangdong/20SF012/2020
   - Guangdong/20SF013/2020
   - Guangdong/20SF014/2020
   - Guangdong/20SF025/2020
   - Guangdong/20SF028/2020
   - Guangdong/20SF040/2020
 * Department of Medical Sciences, National Institute of Health, Nonthaburi, Thailand
   - Nonthaburi/61/2020
   - Nonthaburi/74/2020
 * Division of Viral Diseases, Centers for Disease Control and Prevention, USA
   - USA-WA1/2020

```
