---
title: Análise genômica da propagação do nCoV. Relatório da situação até 25/01/2020.
authors: "Trevor Bedford, Richard Neher, James Hadfield, Emma Hodcroft, Misja Ilcisin, Nicola Müller"
authorLinks: "https://nextstrain.org"
affiliations: "Fred Hutch, Seattle, USA and Biozentrum, Basel, Switzerland"
date: "2020 Jan 25"
dataset: "https://nextstrain.org/ncov/2020-01-25?d=map"
abstract: "Esse relatório usa dados genômicos do novo coronavírus (nCoV) compartilhados publicamente pelo GISAID e Genbank, para estimar as taxas e padrões de propagação da epidemia viral. Planejamos emitir relatórios da situação tão logo dados são produzidos e compartilhados. Esse site é otimizado para visualização em navegadores desktop."
---

# [Resumo](https://nextstrain.org/ncov/2020-01-25)

```auspiceMainDisplayMarkdown
## Resumo

Examinamos a diversidade genética dos 27 genomas do novo coronavírus (nCoV) compartilhados publicamente, a fim de deduzir a data do ancestral comum e a taxa de propagação do vírus.
Concluímos que:
* Os 27 genomas em amostragem são muito similares, acumulando somente entre 0 e 5 mutações
* Essa falta de diversidade genética tem uma explicação cautelosa de que o surto ou descende de uma única introdução do vírus na população humana, ou descende de um pequeno número de transmissões de animais para humanos.
* Esse evento provavelmente ocorreu em Novembro ou no início de Dezembro de 2019.
* Há propagação contínua de humano para humano desde sua introdução, o que resulta nos casos observados.
* Usando o total estimado pela Imperial College London — que aponta milhares de casos —, deduzimos uma taxa de reprodução entre 1,5 e 3,5, o que significa um crescimento rápido no período de novembro a janeiro.
```

# [Coronavírus](https://nextstrain.org/ncov/2020-01-25)

### Leituras adicionais::

* Informações gerais sobre os coronavírus no [Wikipédia](https://pt.wikipedia.org/wiki/Coronavirus) 25/01/2020
* Resumo do surto de nCoV no [Wikipédia](https://pt.wikipedia.org/wiki/Epidemia_de_pneumonia_por_novo_coronav%C3%ADrus_de_2019%E2%80%932020) 25/01/2020
* Material disponibilizado pelo [Centro de Controle e Prevenção de Doenças dos Estados Unidos (CDC)](https://www.cdc.gov/coronavirus/index.html) 23/01/2020
* Características do genoma em [ViralZone](https://viralzone.expasy.org/764?outline=all_by_species) 23/01/2020

```auspiceMainDisplayMarkdown

## Diferentes coronavírus humanos

Os coronavírus (CoV) são membros de um conjunto numeroso de especíes de vírus de RNA de sentido positivo ((+) ssRNA), com histórico de causar infecções respiratórias em humanos.
Alguns tipos de coronavírus estão associados com surtos, já outros estão continuamente circulando e causando principalmente infecções respiratórias leves (como por exemplo, o resfriado comum).

#### SARS-CoV e MERS-CoV
O mais conhecido dos coronavírus é o [SARS-CoV](https://pt.wikipedia.org/wiki/S%C3%ADndrome_respirat%C3%B3ria_aguda_grave) (Síndrome respiratória aguda grave), que foi responsável por um surto global entre novembro de 2002 e julho de 2003, que resultou em [mais de 8000 casos e 774 mortes](https://www.theguardian.com/world/2017/dec/10/sars-virus-bats-china-severe-acute-respiratory-syndrome), com uma taxa de letalidade entre 9 e 11%.

Em 2012, identificaram um novo coronavírus, o [MERS-CoV](https://pt.wikipedia.org/wiki/Coronav%C3%ADrus_da_s%C3%ADndrome_respirat%C3%B3ria_do_Oriente_M%C3%A9dio) ("Síndrome respiratória do Oriente Médio"), causando graves sintomas respiratórios. O MERS resultou em um número de mortes comparável ao SARS, porém sua rota de transmissão é muito diferente. Enquanto o SARS é propagado eficientemente de um humano para outro, infecções humanas com MERS são geralmente resultante de contato com animas, isto é, por camelos (consulte [Dudas _et al._](https://elifesciences.org/articles/31257) para mais informações). O que fez com que o surto de MERS se auto-restringisse a Península Arábica.


#### CoVs Sazonais
Contudo, nem todos os coronavírus são tão mortais quanto o SARS-CoV e MERS-CoV.
Há quatro coronavírus "sazonais" que  normalmente infectam humanos todo ano.
Comparado com o SARS, as cepas sazonais são ["muito mais prevalentes, muito menos graves, e geralmente causas de doenças parecidas com a influenza (ILI, segundo a sigla em inglês)"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5820427/).
Na verdade, [5](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2879166/)–[12](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5820427/)% de todos os casos de ILI testam positivo para coronavírus, portanto os sazonais são muito comuns, o que resulta em milhões de infecções leves todo ano..
Estes coronavírus sazonais são resultados de transbordamentos separados do reservatório de morcegos para humanos no passado (100 anos), em que após a transmissão entre espécies, cada vírus se estabeleceu e se espalhou amplamente na população humana.


#### Reservatórios Animais
Coronavírus infectam uma gama grande de animas, e os surtos humanos descritos anteriormente são resultado de um ou mais "saltos" dos vírus desses reservatórios animais para dentro da população humana.
Acredita-se que o SARS chegou até a população humana por meio de [morcegos-de-ferradura via uma civetas (Paguma larvata) como intermediário](https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1006698).


#### Transmissão de Pessoa para Pessoa
A capacidade de transmitir diferentes linhagens virais entre humanos é extremamente importante para entender o crescimento potencial de um surto.
Devido a sua habilidade de se propagar entre humanos, acrescido de sua alta taxa de letalidade, o SARS (ou um vírus similar a ele) é considerado pela OMS uma [ameaça global  de saúde pública](https://www.who.int/whr/2007/overview/en/index1.html).

```

# [Novel coronavirus (nCoV) 2019-2020](https://nextstrain.org/ncov/2020-01-25)

### Further Reading:

* New China virus: Five questions scientists are asking  [Nature news](https://www.nature.com/articles/d41586-020-00166-6) _2020-01-22_
* China virus latest: first US case confirmed  [Nature news](https://www.nature.com/articles/d41586-020-00154-w) _2020-01-21_
* New virus surging in Asia rattles scientists  [Nature news](https://www.nature.com/articles/d41586-020-00129-x) _2020-01-20_
* New virus identified as likely cause of mystery illness in China [Nature news](https://www.nature.com/articles/d41586-020-00020-9) _2020-01-08_

```auspiceMainDisplayMarkdown

## Recent outbreak of a novel coronavirus
In December 2019, a new illness was first detected in Wuhan, China.
We now know this to be another outbreak of coronavirus in humans (the 7th), and it is provisionally being called nCoV (novel coronavirus).

As of January 25th over 1400 cases and 42 deaths [have been reported](https://en.wikipedia.org/wiki/2019%E2%80%9320_outbreak_of_novel_coronavirus_(2019-nCoV)).
It's still too early to know the case fatality rate, but early indications are that it is significantly less than SARS-CoV.
The case counts are dramatically rising in part due to increased surveillance and testing.

While the outbreak seems to be centered in Wuhan, which is now [under quarantine](https://twitter.com/PDChina/status/1220060879112282117), the virus has spread throughout China and abroad, including Hong Kong, Macau, Thailand, Japan, South Korea, USA, and France. Limited local transmission outside of China has been reported.

The origin of the virus is still unclear, however [genomic analyis](https://virological.org/t/ncovs-relationship-to-bat-coronaviruses-recombination-signals-no-snakes/331) suggests nCoV is most closely related to viruses previously identified in bats.
It is plausible that there were other intermediate animal transmissions before the introduction into humans.
There is no evidence of snakes as an intermediary.

#### Nextstrain narratives

The following pages contain analysis performed using [Nextstrain](https://nextstrain.org).
Scrolling through the left hand sidebar will reveal paragraphs of text with a corresponding visualization of the genomic data on the right hand side.

To have full genomes of a novel and large RNA virus this quickly is a remarkable achievement.
These analyses have been made possible by the rapid and open sharing of genomic data and interpretations by scientists all around the world (see the final slide for a visualization of sequencing authorship).
```

# [How to interpret the phylogenetic trees](https://nextstrain.org/ncov/2020-01-25)

### Further Reading:

* [Exploring interactive phylogenies with Auspice](https://neherlab.org/201901_krisp_auspice.html) _2019-01-24_

```auspiceMainDisplayMarkdown
## Transmission trees vs phylogenetic trees

Pathogens spread through rapid replication in one host followed by transmission to another host.
An epidemic can only take off when one infection results in more than one subsequent infections.

As the pathogen replicates and spreads, its genome needs to be replicated many times and random mutations (copying mistakes)  will accumulate in the genome.
Such random mutations can help to track the spread of the pathogen and learn about its transmission routes and dynamics.

<div>
  <img alt="cartoon showing how transmission tree and phylogenetic tree relate" width="500" src="https://neherlab.org/talk_images/infection_tree_combined.png"/>
</div>

The illustration above shows a sketch of a transmission tree with a subset of cases that were sampled (blue).
In practice, the transmission tree is unknown and typically only rough estimates of case counts are available.
Genome sequences allow us to infer parts of the transmission tree.
In this example, three mutations (little diamonds) are indicated on the tree.
Sequences that have the same mutations are more closely related, so these mutations allow us to group samples into clusters of closely related viruses that belong to the same transmission chains.

### Reading a Phylogenetic Tree

Below, we see an illustration with a phylogenetic tree on the left, where mutations are shown as colored circles. On the right are the corresponding sequences, also with mutations shown as colored circles.
We can see that sequences that share the same mutations group together.
When sequences appear linked by a flat vertical line, like A and B, this means there are no differences between them – their sequences are identical.

When a sequence sits on a long line on its own, like C or E, this means it has unique mutations not found in other sequences. The longer the line, the more mutations.
A and B also have unique mutations (the green circle) not shared by the other sequences, but they are identical to each other.

<div>
  <img alt="cartoon of phylogenetic tree and corresponding alignment, with samples labelled A-E" width="500" src="https://nextstrain-data.s3.amazonaws.com/toy_alignment_tree.png"/>
</div>

At the moment, the novel coronavirus (nCoV) phylogeny may not look much like a 'tree'.
Many of the sequences are identical – they sit together on vertical lines like A and B (some are on the left-most part of the tree).
Others have unique or shared mutations and so sit on lines, or 'branches', going to the right.
You can see how many mutations a branch has by hovering your mouse over it.
```

# [Phylogenetic analysis](http://localhost:4000/ncov/2020-01-25?m=div&d=tree)

Here we present a phylogeny of 27 strains of nCoV that have been publicly shared.
Information on how the analysis was performed is available [in this GitHub repository](github.com/nextstrain/ncov).

<br>

The colours represent the city/US-state of isolation, with the x-axis representing nucleotide divergence.

<br>

Divergence is measured as the number of changes (mutations) in the genome.
Several sequences have zero mutations -- meaning they are all identical to the root (center) of the tree.
Other viruses have between one and five mutations.

<br>

Sequencing the genome of a large novel RNA virus in an evolving outbreak situation is challenging.
Some of the differences observed in these sequences may be sequencing errors rather than actual mutations.
Insertions, deletions, and differences at the ends of the genome are more likely to be errors and so we masked these for the purposes of this analysis.

# [Phylogenetic Interpretation](https://nextstrain.org/ncov/2020-01-25?m=div&d=tree)

We currently see little genetic diversity across the nCoV sequences, with 8 out of 27 sequences having no unique mutations.

<br>

Low genetic diversity across these sequences suggests that the most recent common ancestor of all nCoV sequences was fairly recent, since mutations accumulate slowly compared to other RNA viruses at a rate of around 1-2 mutations per month for coronaviruses.
Generally, repeated introductions from an animal reservoir will show significant diversity (this has been true for Lassa, Ebola, MERS-CoV and avian flu).
The observation of such strong clustering of human infections can be explained by an outbreak that descends from a single zoonotic introduction event into the human population followed by human-to-human epidemic spread.

<br>

At the moment, most mutations that can be observed are singletons – they are unique to individual genomes. Only the sequences that form the two clusters from Guangdong and the sequence form the USA share mutations – we will explore these in later slides.

# [Within-family transmission 1](https://nextstrain.org/2020-01-25/ncov?m=div&d=tree&f_location=Zhuhai)

There are two genetically-identical isolates from Zhuhai (Southeastern China, Guangdong Province) which form a cluster, sharing one unique mutation seen in no other isolate (you can hover your mouse over the branches to see which mutations are present).

<br>

These two cases are [known to come from a single family](https://twitter.com/JingLu_LuJing/status/1220143773532880896), again indicating human-to-human transmission.


# [Within-family transmission 2](https://nextstrain.org/ncov/2020-01-25?m=div&d=tree&f_location=Shenzhen)

Of the six isolates from Shenzhen (Southeastern China, Guangdong Province) we see four isolates which are genetically identical.
These sequences differ by 3 mutations from the root of the tree.
Two of these mutations are also found in the sequence of the case in Washington, USA (more below).
One sequence from Shenzhen has two additional mutations.

<br>

The sequences form Shenzhen are [known to come from a single family](https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(20)30154-9/fulltext), and almost certainly represent human-to-human transmission.

<br>


# [Cases outside China](https://nextstrain.org/ncov/2020-01-25?c=country&d=tree&m=div)

There are reported diagnostically confirmed nCoV cases in many East and South-East Asian countries, USA, Australia, and Europe.
Vietnam has reported a case without travel history to Wuhan who possibly got infected from a [relative visiting from Wuhan](https://www.aljazeera.com/news/2020/01/china-coronavirus-outbreak-latest-updates-200123133559818.html).

<br>

The only currently available sequence data for cases outside of China are the two cases from Thailand and one case from the USA, which are coloured here in yellow and red, respectively.
The Thai samples are genetically identical to six Chinese sequences, including five isolated in Wuhan.
The sequence from Washington, US, shares two mutations with the cluster of sequences from Shenzhen.

<br>

The case from Washington, USA, had travelled to Wuhan.
The most parsimonious explanation for the observed pattern of mutation sharing is the virus variant with the two shared mutations was circulating in Wuhan and was independently exported to Shenzhen and to the USA.


# [Dating the time of the most recent common ancestor](https://nextstrain.org/ncov/2020-01-25?d=tree)
The high similarity of the genomes suggests they share a recent common ancestor (i.e. that they have descended from the same ancestral virus recently). Otherwise, we would expect a higher number of differences between the samples.

<br>

Previous research on related coronavirus suggests that these viruses accumulate between 1 and 3 changes in their genome per month (rates of 3 &times; 10<sup>-4</sup> to 1 &times; 10<sup>-3</sup> per site per year).

<br>

On the right, we explore how different assumptions about the rate of change, and the observed genetic diversity, give us estimates for the timing of the outbreak.

```auspiceMainDisplayMarkdown
## Date of the common ancestor of outbreak viruses
Here, we assume a star-like phylogeny structure along with a Poisson distribution of mutations through time to estimate the time of the most recent common ancestor ('TMRCA') of sequenced viruses.
**We find that the common ancestor most likely existed between mid-Nov and the beginning of Dec 2019.**

<div>
  <img alt="graph of TMRCA estimates based on different mutation rates" width="500" src="https://nextstrain-data.s3.amazonaws.com/ncov_poisson-tmrca.png"/>
</div>

As the more samples are sequenced, we expect the tree to show more structure, such that the star-like phylogeny topology is no longer a good assumption.
At this point, phylodynamic estimates of the age of the epidemic will become feasible.

There is a [confirmed case in Wuhan with onset date of Dec 1, 2019](https://twitter.com/trvrb/status/1220749265380593664), which would put an upper bound on the date of most recent common ancestor. And strongly suggests evolutionary rates of ~3.0 x 10^-4 are more plausible.

```

# [Estimating the growth rate](https://nextstrain.org/ncov/2020-01-25?d=tree)

An important quantity in the spread of a pathogen is the average number of secondary cases each infection produces.

<br>

This number is known as R0 ("R-zero" or "R-nought").
One the right, we present simple estimates of R0.

```auspiceMainDisplayMarkdown
## Estimates of epidemic growth rate
Scientists at Imperial College London have used the number of cases observed outside of China to estimate the [total number of cases](https://www.imperial.ac.uk/mrc-global-infectious-disease-analysis/news--wuhan-coronavirus/) and suggested that there have been at least several thousand cases.
Together with our previous estimates of the age of the outbreak and information on the infectious period, we can estimate plausible ranges of R0 using a branching process model.

**We find plausible estimates of R0 between 1.5 and 3.5.**

If we assume the outbreak started at the beginning of November 2019 (12 weeks ago), we find that R0 should range between 1.5 and 2.5, depending on how large ('n') the outbreak is now.
<div>
  <img alt="graph of R0 estimates with epidemic start 12 weeks ago" width="500" src="https://nextstrain-data.s3.amazonaws.com/ncov_branching-R0-early.png"/>
</div>

If we assume a more recent start, at the beginning of December 2019 (8 weeks ago), the estimates for R0 range between 1.8 and 3.5:
<div>
  <img alt="graph of R0 estimates with epidemic start 8 weeks ago" width="500" src="https://nextstrain-data.s3.amazonaws.com/ncov_branching-R0-recent.png"/>
</div>
```

# [Scientific credit](https://nextstrain.org/ncov/2020-01-25?d=map&c=author)

We would like to acknowledge the amazing and timely work done by all scientists involved in this outbreak, but particularly those working in China.
Only through the rapid sharing of genomic data and metadata are analyses such as these possible.

<br>

The nCoV genomes were generously shared by scientists at the:

 * Shanghai Public Health Clinical Center & School of Public Health, Fudan University, Shanghai, China
 * National Institute for Viral Disease Control and Prevention, China CDC, Beijing, China
 * Institute of Pathogen Biology, Chinese Academy of Medical Sciences & Peking Union Medical College, Beijing, China
 * Wuhan Institute of Virology, Chinese Academy of Sciences, Wuhan, China
 * Department of Microbiology, Zhejiang Provincial Center for Disease Control and Prevention, Hangzhou, China
 * Guangdong Provincial Center for Diseases Control and Prevention
 * Department of Medical Sciences, National Institute of Health, Nonthaburi, Thailand
 * Division of Viral Diseases, Centers for Disease Control and Prevention, USA

# [Detailed scientific credit](https://nextstrain.org/ncov/2020-01-25?d=map&c=author)

These data were shared via [GISAID](https://gisaid.org).
We gratefully acknowledge their contributions.

<br>

To the right we give specific sequences shared by each lab.

```auspiceMainDisplayMarkdown

The nCoV genomes were generously shared by scientists at the

 * Shanghai Public Health Clinical Center & School of Public Health, Fudan University, Shanghai, China
   - Wuhan-Hu-1/2019
 * National Institute for Viral Disease Control and Prevention, China CDC, Beijing, China
   - Wuhan/IVDC-HB-01/2019
   - Wuhan/IVDC-HB-04/2020
   - Wuhan/IVDC-HB-05/2019)
 * Institute of Pathogen Biology, Chinese Academy of Medical Sciences & Peking Union Medical College, Beijing, China
   - Wuhan/IPBCAMS-WH-01/2019
   - Wuhan/IPBCAMS-WH-02/2019
   - Wuhan/IPBCAMS-WH-03/2019
   - Wuhan/IPBCAMS-WH-04/2019
 * Wuhan Institute of Virology, Chinese Academy of Sciences, Wuhan, China
   - Wuhan/WIV02/2019
   - Wuhan/WIV04/2019
   - Wuhan/WIV05/2019
   - Wuhan/WIV06/2019
   - Wuhan/WIV07/2019
 * Department of Microbiology, Zhejiang Provincial Center for Disease Control and Prevention, Hangzhou, China
   - Zhejiang/WZ-01/2020
   - Zhejiang/WZ-02/2020
 * Guangdong Provincial Center for Diseases Control and Prevention
   - Guangdong/20SF012/2020
   - Guangdong/20SF013/2020
   - Guangdong/20SF014/2020
   - Guangdong/20SF025/2020
   - Guangdong/20SF028/2020
   - Guangdong/20SF040/2020
 * Department of Medical Sciences, National Institute of Health, Nonthaburi, Thailand
   - Nonthaburi/61/2020
   - Nonthaburi/74/2020
 * Division of Viral Diseases, Centers for Disease Control and Prevention, USA
   - USA-WA1/2020

```
