---
title: Análisis genómico de la propagación de nCoV. Informe de la situación hasta 2020-01-25.
authors: "Trevor Bedford, Richard Neher, James Hadfield, Emma Hodcroft, Misja Ilcisin, Nicola Müller, Julian Villabona-Arenas"
authorLinks: "https://nextstrain.org"
affiliations: "Fred Hutch, Seattle, USA; Biozentrum, Basel, Switzerland; and CMMID & LSHTM, London, UK"
date: "2020 Jan 25"
dataset: "https://nextstrain.org/ncov/2020-01-25?d=map"
abstract: " Este informe utiliza datos genómicos del nuevo coronavirus (nCoV) compartidos públicamente en GISAID y Genbank para estimar las tasas y los patrones de propagación de la epidemia viral. Planeamos emitir informes de la situación que se actualizaran en la medida que se produzcan y compartan nuevos datos. Este sitio web está optimizado para su visualización en navegadores web de escritorio."

traducción: "Julian Villabona-Arenas"
afiliación: "CMMID & LSHTM, London, UK"
fecha de la traducción: "2020 Jan 26"

---

# [Resumen Ejecutivo](https://nextstrain.org/ncov/2020-01-25)

```auspiceMainDisplayMarkdown
## Resumen Ejecutivo

Examinamos la diversidad genética 27 genomas públicos del nuevo coronavirus (nCoV) para inferir la fecha del ancestro común más reciente y la tasa de propagación.
Encontramos:
* Los 27 genomas muestreados son muy similares y solo han acumulado entre 0 y 5 mutaciones.
* La explicación mas parsimoniosa para este bajo grado de diversidad genética es que el brote proviene de una sola introducción en la población humana. También es posible que un pequeño número de transmisiones de virus muy similares hayan ocurrido desde animales a humanos.
* Este evento probablemente ocurrió en noviembre o principios de diciembre de 2019.
* Ha habido una propagación continua de persona a persona desde su introducción, lo que resulta en los casos observados.
* Usando las estimaciones del número total de casos hechas por el Imperial College de Londres (del orden de varios miles de casos), inferimos un número reproductivo que oscila entre 1.5 y 3.5. Estas cifras indican un rápido crecimiento de la epidemia en el período de noviembre a enero.
```

# [Coronavirus](https://nextstrain.org/ncov/2020-01-25)

### Lecturas adicionales:

* Información general sobre los coronavirus en [Wikipedia](https://es.wikipedia.org/wiki/Coronavirus) _2020-01-25_
* Resumen del brote por nCov en [Wikipedia](https://es.wikipedia.org/wiki/Epidemia_de_neumon%C3%ADa_por_coronavirus_de_Wuhan_de_2019-2020) _2020-01-25_
* Material proporcionado por los Centros para el Control y la Prevención de Enfermedades de los Estados Unidos en [US CDC](https://www.cdc.gov/coronavirus/index.html) _2020-01-23_
* Características del genoma en [ViralZone](https://viralzone.expasy.org/764?outline=all_by_species) _2020-01-23_

```auspiceMainDisplayMarkdown

## Diferentes coronavirus humanos


Los coronavirus (CoV) agrupan diversas especies de virus de ARN ((+) ssRNA) de sentido positivo que causan infecciones respiratorias en humanos.
Algunas variantes de los coronavirus están asociadas solamente con brotes, otras circulan continuamente y causan generalmente infecciones respiratorias leves (por ejemplo, el resfriado común).

#### SARS-CoV y MERS-CoV
El más conocido de estos coronavirus es el [SARS-CoV](https://es.wikipedia.org/wiki/S%C3%ADndrome_respiratorio_agudo_grave) ("síndrome respiratorio agudo severo"), que en un brote que ocurrió entre noviembre de 2002 y julio de 2003 y se propagó por todo el mundo resultando en [más de 8000 casos y 774 muertes](https://www.theguardian.com/world/2017/dec/10/sars-virus-bats-china-severe-acute-respiratory-syndrome), con una tasa de letalidad de alrededor del 9-11%.

En 2012, un nuevo coronavirus [MERS-CoV](https://en.wikipedia.org/wiki/Middle_East_respiratory_syndrome) ("Síndrome respiratorio del Medio Oriente"), asociado con síntomas respiratorios graves fue identificado. MERS ha resultado en un número de muertes comparable al SARS, sin embargo, la ruta de transmisión de MERS es muy diferente. Mientras que el SARS se propagó eficientemente de un humano a otro, las infecciones por MERS en humanos fueron generalmente el resultado de zoonosis independientes (transmisiones de animales a humanos) a partir de  camellos (consultar [Dudas _et al._](https://elifesciences.org/articles/31257) para mayores informaciones). Esto ha llevado a que el brote de MERS se limita a la Península Arábiga.


#### CoV estacional
Sin embargo, no todos los coronavirus son tan mortales como el SARS-CoV y el MERS-CoV. Hay cuatro coronavirus "estacionales" que comúnmente infectan a los humanos cada año.
En comparación con el SARS, estas cepas estacionales de coronavirus son ["mucho más prevalentes, mucho menos graves y causan generalmente enfermedades similares a la influenza (ILI)"](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5820427/).
De hecho, [5](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2879166/)–[12](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5820427/)% de todos los casos de Ili dan positivo en las pruebas de infección por coronavirus, por lo tanto, son bastante comunes, lo que resulta en millones de infecciones leves cada año.
Estos coronavirus estacionales son el resultado de múltiples zoonosis a partir de murciélagos que ocurrieron hace aproximadamente 100 años. Después de la transmisión entre especies, cada virus estableció una transmisión continua y generalizada en la población humana.


#### Reservorios animales
Los coronavirus infectan una amplia gama de animales y los brotes humanos supracitados son el resultado de uno o más "saltos" de estos reservorios animales a la población humana.
Se cree que el SARS fue introducido en la población humana a partir de  [murciélagos de herradura--Rhinolophidae--con la ayuda de civetas--Paguma larvata--como intermediario](https://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1006698).


#### Transmisión de persona a persona
La capacidad de transmitir diferentes linajes virales entre humanos es extremadamente importante para comprender el potencial desarrollo de un brote.
Debido a la capacidad del SARS de propagarse entre los humanos y su alta tasa de letalidad, el SARS (o cualquier virus similar al SARS) se considera una [amenaza de salud pública global](https://www.who.int/whr/2007/overview/en/index1.html) por la OMS.

```

# [Nuevo coronavirus (nCoV) 2019-2020](https://nextstrain.org/ncov/2020-01-25)

### Lecturas adicionales:

* Nuevo virus en China: cinco preguntas que se hacen los científicos (en inglés)  [Nature news](https://www.nature.com/articles/d41586-020-00166-6) _2020-01-22_
* Última actualización del virus en China: primer caso confirmado en los Estados Unidos (en inglés)   [Nature news](https://www.nature.com/articles/d41586-020-00154-w) _2020-01-21_
* Un nuevo virus en Asia preocupa a los científicos (en inglés)  [Nature news](https://www.nature.com/articles/d41586-020-00129-x) _2020-01-20_
* Un nuevo virus es identificado como la causa probable de la enfermedad misteriosa en China (en inglés) [Nature news](https://www.nature.com/articles/d41586-020-00020-9) _2020-01-08_

```auspiceMainDisplayMarkdown

## Brote reciente por un nuevo coronavirus
En diciembre de 2019, se detectó por primera vez una nueva enfermedad en Wuhan, China.
Actualmente sabemos que se trata de un nuevo brote por coronavirus en humanos (el séptimo), y se llama provisionalmente nCoV (nuevo coronavirus).


Hasta el 25 de enero, más de 1400 casos y 42 muertes [han sido reportados](https://en.wikipedia.org/wiki/2019%E2%80%9320_outbreak_of_novel_coronavirus_(2019-nCoV)).
Todavía es demasiado pronto para saber la tasa de letalidad, pero los datos preliminares indican que es significativamente menor que el SARS-CoV.
El número de casos aumenta rápidamente en parte debido a una mayor vigilancia epidemiologica y un aumento de las pruebas de diagnóstico.

Si bien el brote parece estar centrado en Wuhan, que ahora está [en cuarentena](https://twitter.com/PDChina/status/1220060879112282117), el virus se ha extendido por toda China y en el exterior, incluyendo Hong Kong, Macao, Tailandia, Japón, Corea del Sur, Estados Unidos y Francia. La transmisión local fuera de China ha sido limitada hasta el momento.

El origen del virus aún no está claro, sin embargo [análisis genómicos](https://virological.org/t/ncovs-relationship-to-bat-coronaviruses-recombination-signals-no-snakes/331) sugiere que nCoV está más estrechamente relacionado con virus previamente identificados en murciélagos.
Es posible que la transmissión haya ocurrido a traves de otros animales (intermediarios) antes de la introducción en humanos.
No hay evidencia de serpientes como intermediario.

#### Narrativas Nextstrain 

Las siguientes páginas contienen análisis realizados con [Nextstrain](https://nextstrain.org).
El texto aparecerá al desplazarse con la barra lateral izquierda y la visualización de los datos genómicos correspondiente será mostrada en el lado derecho.

La obtención (secuenciación) rápida de genomas completos de un nuevo virus de ARN es un logro notable.
Estos análisis han sido posibles gracias al intercambio rápido y abierto de datos genómicos y a las interpretaciones por parte de científicos de todo el mundo (la diapositiva final incluye los autores de cada uno de los diferentes esfuerzos de secuenciación).
```

# [How to interpret the phylogenetic trees](https://nextstrain.org/ncov/2020-01-25)

### Further Reading:

* [Exploring interactive phylogenies with Auspice](https://neherlab.org/201901_krisp_auspice.html) _2019-01-24_

```auspiceMainDisplayMarkdown
## Transmission trees vs phylogenetic trees

Pathogens spread through rapid replication in one host followed by transmission to another host.
An epidemic can only take off when one infection results in more than one subsequent infections.

As the pathogen replicates and spreads, its genome needs to be replicated many times and random mutations (copying mistakes)  will accumulate in the genome.
Such random mutations can help to track the spread of the pathogen and learn about its transmission routes and dynamics.

<div>
  <img alt="cartoon showing how transmission tree and phylogenetic tree relate" width="500" src="https://neherlab.org/talk_images/infection_tree_combined.png"/>
</div>

The illustration above shows a sketch of a transmission tree with a subset of cases that were sampled (blue).
In practice, the transmission tree is unknown and typically only rough estimates of case counts are available.
Genome sequences allow us to infer parts of the transmission tree.
In this example, three mutations (little diamonds) are indicated on the tree.
Sequences that have the same mutations are more closely related, so these mutations allow us to group samples into clusters of closely related viruses that belong to the same transmission chains.

### Reading a Phylogenetic Tree

Below, we see an illustration with a phylogenetic tree on the left, where mutations are shown as colored circles. On the right are the corresponding sequences, also with mutations shown as colored circles.
We can see that sequences that share the same mutations group together.
When sequences appear linked by a flat vertical line, like A and B, this means there are no differences between them – their sequences are identical.

When a sequence sits on a long line on its own, like C or E, this means it has unique mutations not found in other sequences. The longer the line, the more mutations.
A and B also have unique mutations (the green circle) not shared by the other sequences, but they are identical to each other.

<div>
  <img alt="cartoon of phylogenetic tree and corresponding alignment, with samples labelled A-E" width="500" src="https://nextstrain-data.s3.amazonaws.com/toy_alignment_tree.png"/>
</div>

At the moment, the novel coronavirus (nCoV) phylogeny may not look much like a 'tree'.
Many of the sequences are identical – they sit together on vertical lines like A and B (some are on the left-most part of the tree).
Others have unique or shared mutations and so sit on lines, or 'branches', going to the right.
You can see how many mutations a branch has by hovering your mouse over it.
```

# [Phylogenetic analysis](http://localhost:4000/ncov/2020-01-25?m=div&d=tree)

Here we present a phylogeny of 27 strains of nCoV that have been publicly shared.
Information on how the analysis was performed is available [in this GitHub repository](github.com/nextstrain/ncov).

<br>

The colours represent the city/US-state of isolation, with the x-axis representing nucleotide divergence.

<br>

Divergence is measured as the number of changes (mutations) in the genome.
Several sequences have zero mutations -- meaning they are all identical to the root (center) of the tree.
Other viruses have between one and five mutations.

<br>

Sequencing the genome of a large novel RNA virus in an evolving outbreak situation is challenging.
Some of the differences observed in these sequences may be sequencing errors rather than actual mutations.
Insertions, deletions, and differences at the ends of the genome are more likely to be errors and so we masked these for the purposes of this analysis.

# [Phylogenetic Interpretation](https://nextstrain.org/ncov/2020-01-25?m=div&d=tree)

We currently see little genetic diversity across the nCoV sequences, with 8 out of 27 sequences having no unique mutations.

<br>

Low genetic diversity across these sequences suggests that the most recent common ancestor of all nCoV sequences was fairly recent, since mutations accumulate slowly compared to other RNA viruses at a rate of around 1-2 mutations per month for coronaviruses.
Generally, repeated introductions from an animal reservoir will show significant diversity (this has been true for Lassa, Ebola, MERS-CoV and avian flu).
The observation of such strong clustering of human infections can be explained by an outbreak that descends from a single zoonotic introduction event into the human population followed by human-to-human epidemic spread.

<br>

At the moment, most mutations that can be observed are singletons – they are unique to individual genomes. Only the sequences that form the two clusters from Guangdong and the sequence form the USA share mutations – we will explore these in later slides.

# [Within-family transmission 1](https://nextstrain.org/2020-01-25/ncov?m=div&d=tree&f_location=Zhuhai)

There are two genetically-identical isolates from Zhuhai (Southeastern China, Guangdong Province) which form a cluster, sharing one unique mutation seen in no other isolate (you can hover your mouse over the branches to see which mutations are present).

<br>

These two cases are [known to come from a single family](https://twitter.com/JingLu_LuJing/status/1220143773532880896), again indicating human-to-human transmission.


# [Within-family transmission 2](https://nextstrain.org/ncov/2020-01-25?m=div&d=tree&f_location=Shenzhen)

Of the six isolates from Shenzhen (Southeastern China, Guangdong Province) we see four isolates which are genetically identical.
These sequences differ by 3 mutations from the root of the tree.
Two of these mutations are also found in the sequence of the case in Washington, USA (more below).
One sequence from Shenzhen has two additional mutations.

<br>

The sequences form Shenzhen are [known to come from a single family](https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(20)30154-9/fulltext), and almost certainly represent human-to-human transmission.

<br>


# [Cases outside China](https://nextstrain.org/ncov/2020-01-25?c=country&d=tree&m=div)

There are reported diagnostically confirmed nCoV cases in many East and South-East Asian countries, USA, Australia, and Europe.
Vietnam has reported a case without travel history to Wuhan who possibly got infected from a [relative visiting from Wuhan](https://www.aljazeera.com/news/2020/01/china-coronavirus-outbreak-latest-updates-200123133559818.html).

<br>

The only currently available sequence data for cases outside of China are the two cases from Thailand and one case from the USA, which are coloured here in yellow and red, respectively.
The Thai samples are genetically identical to six Chinese sequences, including five isolated in Wuhan.
The sequence from Washington, US, shares two mutations with the cluster of sequences from Shenzhen.

<br>

The case from Washington, USA, had travelled to Wuhan.
The most parsimonious explanation for the observed pattern of mutation sharing is the virus variant with the two shared mutations was circulating in Wuhan and was independently exported to Shenzhen and to the USA.


# [Dating the time of the most recent common ancestor](https://nextstrain.org/ncov/2020-01-25?d=tree)
The high similarity of the genomes suggests they share a recent common ancestor (i.e. that they have descended from the same ancestral virus recently). Otherwise, we would expect a higher number of differences between the samples.

<br>

Previous research on related coronavirus suggests that these viruses accumulate between 1 and 3 changes in their genome per month (rates of 3 &times; 10<sup>-4</sup> to 1 &times; 10<sup>-3</sup> per site per year).

<br>

On the right, we explore how different assumptions about the rate of change, and the observed genetic diversity, give us estimates for the timing of the outbreak.

```auspiceMainDisplayMarkdown
## Date of the common ancestor of outbreak viruses
Here, we assume a star-like phylogeny structure along with a Poisson distribution of mutations through time to estimate the time of the most recent common ancestor ('TMRCA') of sequenced viruses.
**We find that the common ancestor most likely existed between mid-Nov and the beginning of Dec 2019.**

<div>
  <img alt="graph of TMRCA estimates based on different mutation rates" width="500" src="https://nextstrain-data.s3.amazonaws.com/ncov_poisson-tmrca.png"/>
</div>

As the more samples are sequenced, we expect the tree to show more structure, such that the star-like phylogeny topology is no longer a good assumption.
At this point, phylodynamic estimates of the age of the epidemic will become feasible.

There is a [confirmed case in Wuhan with onset date of Dec 1, 2019](https://twitter.com/trvrb/status/1220749265380593664), which would put an upper bound on the date of most recent common ancestor. And strongly suggests evolutionary rates of ~3.0 x 10^-4 are more plausible.

```

# [Estimating the growth rate](https://nextstrain.org/ncov/2020-01-25?d=tree)

An important quantity in the spread of a pathogen is the average number of secondary cases each infection produces.

<br>

This number is known as R0 ("R-zero" or "R-nought").
One the right, we present simple estimates of R0.

```auspiceMainDisplayMarkdown
## Estimates of epidemic growth rate
Scientists at Imperial College London have used the number of cases observed outside of China to estimate the [total number of cases](https://www.imperial.ac.uk/mrc-global-infectious-disease-analysis/news--wuhan-coronavirus/) and suggested that there have been at least several thousand cases.
Together with our previous estimates of the age of the outbreak and information on the infectious period, we can estimate plausible ranges of R0 using a branching process model.

**We find plausible estimates of R0 between 1.5 and 3.5.**

If we assume the outbreak started at the beginning of November 2019 (12 weeks ago), we find that R0 should range between 1.5 and 2.5, depending on how large ('n') the outbreak is now.
<div>
  <img alt="graph of R0 estimates with epidemic start 12 weeks ago" width="500" src="https://nextstrain-data.s3.amazonaws.com/ncov_branching-R0-early.png"/>
</div>

If we assume a more recent start, at the beginning of December 2019 (8 weeks ago), the estimates for R0 range between 1.8 and 3.5:
<div>
  <img alt="graph of R0 estimates with epidemic start 8 weeks ago" width="500" src="https://nextstrain-data.s3.amazonaws.com/ncov_branching-R0-recent.png"/>
</div>
```

# [Scientific credit](https://nextstrain.org/ncov/2020-01-25?d=map&c=author)

We would like to acknowledge the amazing and timely work done by all scientists involved in this outbreak, but particularly those working in China.
Only through the rapid sharing of genomic data and metadata are analyses such as these possible.

<br>

The nCoV genomes were generously shared by scientists at the:

 * Shanghai Public Health Clinical Center & School of Public Health, Fudan University, Shanghai, China
 * National Institute for Viral Disease Control and Prevention, China CDC, Beijing, China
 * Institute of Pathogen Biology, Chinese Academy of Medical Sciences & Peking Union Medical College, Beijing, China
 * Wuhan Institute of Virology, Chinese Academy of Sciences, Wuhan, China
 * Department of Microbiology, Zhejiang Provincial Center for Disease Control and Prevention, Hangzhou, China
 * Guangdong Provincial Center for Diseases Control and Prevention
 * Department of Medical Sciences, National Institute of Health, Nonthaburi, Thailand
 * Division of Viral Diseases, Centers for Disease Control and Prevention, USA

# [Detailed scientific credit](https://nextstrain.org/ncov/2020-01-25?d=map&c=author)

These data were shared via [GISAID](https://gisaid.org).
We gratefully acknowledge their contributions.

<br>

To the right we give specific sequences shared by each lab.

```auspiceMainDisplayMarkdown

The nCoV genomes were generously shared by scientists at the

 * Shanghai Public Health Clinical Center & School of Public Health, Fudan University, Shanghai, China
   - Wuhan-Hu-1/2019
 * National Institute for Viral Disease Control and Prevention, China CDC, Beijing, China
   - Wuhan/IVDC-HB-01/2019
   - Wuhan/IVDC-HB-04/2020
   - Wuhan/IVDC-HB-05/2019)
 * Institute of Pathogen Biology, Chinese Academy of Medical Sciences & Peking Union Medical College, Beijing, China
   - Wuhan/IPBCAMS-WH-01/2019
   - Wuhan/IPBCAMS-WH-02/2019
   - Wuhan/IPBCAMS-WH-03/2019
   - Wuhan/IPBCAMS-WH-04/2019
 * Wuhan Institute of Virology, Chinese Academy of Sciences, Wuhan, China
   - Wuhan/WIV02/2019
   - Wuhan/WIV04/2019
   - Wuhan/WIV05/2019
   - Wuhan/WIV06/2019
   - Wuhan/WIV07/2019
 * Department of Microbiology, Zhejiang Provincial Center for Disease Control and Prevention, Hangzhou, China
   - Zhejiang/WZ-01/2020
   - Zhejiang/WZ-02/2020
 * Guangdong Provincial Center for Diseases Control and Prevention
   - Guangdong/20SF012/2020
   - Guangdong/20SF013/2020
   - Guangdong/20SF014/2020
   - Guangdong/20SF025/2020
   - Guangdong/20SF028/2020
   - Guangdong/20SF040/2020
 * Department of Medical Sciences, National Institute of Health, Nonthaburi, Thailand
   - Nonthaburi/61/2020
   - Nonthaburi/74/2020
 * Division of Viral Diseases, Centers for Disease Control and Prevention, USA
   - USA-WA1/2020

```
